### ISSUE TITLE
Enhancement Request: Need markdown preview support MathJax

### ISSUE THREAD

**Issue by snyang** [Microsoft/vscode#24070]
- VSCode Version: Code 1.10.2 (8076a19fdcab7e1fc1707952d652f0bb6c6db331, 2017-03-08T14:02:52.799Z)
- OS Version: Windows_NT ia32 6.1.7601
Hope markdown preview supports custom javascripts so that display the following LaTex scripts in well format.

$$
\tanh(x) = \frac{e^{2x} - 1}{e^{2x} + 1} \\
tanh'(x) = 1 - tanh(x)^2
$$

If allow to add following scripts in the preview html, that's would help.

    <script type="text/x-mathjax-config">
        MathJax.Hub.Config({
            tex2jax: {
            inlineMath: [['$','$'], ['\\(','\\)']],
            processEscapes: true,
            showProcessingMessages: false
            },
            displayAlign : "left",
            TeX: {
                        equationNumbers: {
                            // autoNumber: "all",
                            useLabelIds: true
                        }
                  },
                    "HTML-CSS": {
                        linebreaks: {
                            automatic: true
                        },
                        scale: 80,
                        styles: {
                        ".MathJax_Display": {
                            "text-align": "left",
                            "width" : "auto",
                            "margin": "10px 10px 10px 0px !important",
                            "background-color": "#f5f5f5 !important",
                            "border-radius": "3px !important",
                            border:  "1px solid #ccc !important",
                            padding: "5px 5px 5px 5px !important"
                        },
                        ".MathJax": {
                            "background-color": "#f5f5f5 !important",
                            padding: "2px 2px 2px 2px !important"
                        }
                        }
                    },
                    SVG: {
                        linebreaks: {
                            automatic: true
                        }
                    }
            });
    </script>
    <script type="text/javascript" async src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-MML-AM_HTMLorMML">
    </script>

**Comment by mjbvz** [comment]
Duplicate of #6529
